### PR TITLE
update roamjs-components to 0.80.3

### PR DESCRIPTION
**Pull request by mdroidian**
I do not have access to the dropbox repo to check for repo secrets @dvargas92495

### CODE REVIEW

**Comment by mdroidian** [comment]
Hmm... looks like `npm start` is still creating an `index.js` in root.

**Comment by mdroidian** [comment]
~~The other thing that is still different is that this is still on v2. (query builder is on v3)~~

https://github.com/RoamJS/dropbox/blob/ba9cddfe57b80ef48668ca3162c259b96ed2e64c/.github/workflows/main.yaml#L20

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: install
+        run: npm install
+      - name: build
+        run: npx samepage build
```
Comment (dvargasfuertes) [inline .github/workflows/pr.yaml:17]: add the `index.js`/`extension.js` below to the `.gitignore` and remove from the PR

**Review by dvargasfuertes — COMMENTED** [review]
check `tsconfig.json` and make sure it's pointing to the right file

**Pull request merged**